.gitmodules

```diff
@@ -1,9 +1,11 @@
 [submodule "spack"]
   path = spack
-  #url = https://github.com/spack/spack
-  #branch = develop
-  url = https://github.com/NOAA-EMC/spack
-  branch = jcsda_emc_spack_stack
+  ##url = https://github.com/spack/spack
+  ##branch = develop
+  #url = https://github.com/NOAA-EMC/spack
+  #branch = jcsda_emc_spack_stack
+  url = https://github.com/climbfuji/spack
```

Comment:
Is this intentional? Seems like you are switching to your fork here?

Reply:
That's needed for the CI tests to work properly. Until the spack PR is merged, the spack-stack submodule pointer must point to the branch and fork to find the updated code. Procedure is: CI tests need to complete, then we merge the spack PR (pending approval, of course), then update the `spack` submodule pointer in spack-stack and revert `.gitmodules`. Then CI runs again, but can be bypassed if the code in the spack PR is identical with the final code in our (EMC) spack fork/branch - this is normally the case and can be assessed quickly (I do that after merging the spack PR).

Reply:
BTW, the ufs-weather-model is using the same for its code commit procedures.